bazel/linux/utils.bzl

```diff
@@ -117,5 +117,6 @@ The modules still to expand are:
         if key in dups:
             continue
         result.append(mod)
+        dups|={key : ""}
```

Comment:
good catch! so, starlark is not quite like python (would have used a set() instead),  I've never used this syntax. How about a simple `dups[key] = True` for example?

Reply:
Changed. 
Picked `dict` mostly because `dups` was introduced as a dict a few lines above (and has an O(1) lookup time, if we ever expect this list to grow long).